ArcGISRuntimeSDKQt_QMLSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.qml

```diff
@@ -110,7 +133,17 @@ Rectangle {
             identifiedFeature = result.geoElements[0];
             utilityNetworkSource = utilityNetwork.definition.networkSource(identifiedFeature.featureTable.tableName);
 
+            // get domain network
+            domainNetwork = utilityNetwork.definition.domainNetwork("ElectricDistribution");
+
+            const tiers = domainNetwork.tiers;
+            for (let i = 0; i < tiers.length; i++) {
+                if ( tiers[i].name === "Medium Voltage Radial" )
+                    mediumVoltageTier = tiers[i];
+            }
+
             if (utilityNetworkSource.sourceType === Enums.UtilityNetworkSourceTypeJunction) {
+                juncSelected = true;
```

Comment:
Same comment as C++ - optional, but `junctionSelected` is not many more letters, and clearer to me.